packages/editor/src/client/atoms/dateTimePicker.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import {DatePicker, FormGroup, ControlLabel, Form} from 'rsuite'
+import {useTranslation} from 'react-i18next'
+
+export interface DateTimeProps {
+  dateTime?: Date
+  label: string
+  changeDate(publishDate?: Date): void
+}
+
+export function DateTimePicker({dateTime, label, changeDate}: DateTimeProps) {
+  const {t} = useTranslation()
+
+  return (
+    <Form fluid={true}>
+      <FormGroup>
+        <ControlLabel>{label}</ControlLabel>
+        <DatePicker
+          style={{marginRight: 8}}
+          placement="auto"
+          value={dateTime?.getTime() === dateTime?.getTime() ? dateTime : undefined}
+          format="DD MMM YYYY"
+          ranges={[
```

Comment:
I would make it possible to add its own ranges in the `DateTimeProps` but make it optional. If no ranges have been provider we use those as defaults. But make sure that the array passed in `DateTimeProps` is also types.

```
export interface DateTimeRange {
  label: string
  value: (): void
}

export interface DateTimeProps {
  ...
  dayRanges?: DateTimeRange[]
  timeRanges?: DatetimeRange[]
 ...
}
```